### DISCUSSION TITLE
Troubleshoot documentation for Linkerd2 is inconsistant - No such controller pod anymore

### DISCUSSION THREAD

**Discussion by spstarr** [linkerd/linkerd2#8764] [General] [answered]
So, while looking for another issue, having fixed that. I discovered the troubleshooting documentation for linkerd2 2.10/2.11.x is inconsistant:

specifically:

```
√ controller pod is running
Example failure:

× controller pod is running
    No running pods for "linkerd-controller"
Note, it takes a little bit for pods to be scheduled, images to be pulled and everything to start up. If this is a permanent error, you’ll want to validate the state of the controller pod with:

$ kubectl -n linkerd get po --selector linkerd.io/control-plane-component=controller
NAME                                  READY     STATUS    RESTARTS   AGE
linkerd-controller-7bb8ff5967-zg265   4/4       Running   0          40m
Check the controller’s logs with:

kubectl logs -n linkerd linkerd-controller-7bb8ff5967-zg265 public-api
```

When running `linkerd check` there is no such output, closest being:

```
linkerd-existence
-----------------
√ 'linkerd-config' config map exists
√ heartbeat ServiceAccount exist
√ control plane replica sets are ready
√ no unschedulable pods
√ control plane pods are ready
...
linkerd-control-plane-proxy
---------------------------
√ control plane proxies are healthy
√ control plane proxies are up-to-date
√ control plane proxies and cli versions match
```
So, can the documentation be updated to reflect there is no controller pods anymore?

In the `linkerd` namespace: linkerd-destination [linkerd-proxy destination sp-validator policy], linkerd-identity [identity linkerd-proxy], linked-proxy-injector [linkerd-proxy proxy-injector].

Given there's no errors in `linkerd check` I suspect this documentation is stale.

Thanks.
Shawn

**Comment by mateiidavid** [comment, marked as answer]
@spstarr hey, thanks for raising this! I put up a PR to remove any mentions of the controller in our troubleshooting doc page :) 

https://github.com/linkerd/website/pull/1374

**Comment by spstarr** [comment]
PR fixes documentation, thanks so much!
